cdp_scrapers/legistar_utils.py

```diff
@@ -393,8 +402,14 @@ def get_events(
             begin=begin,
             end=end,
         ):
-            session_time = self.date_time_to_datetime(
-                legistar_ev[LEGISTAR_SESSION_DATE], legistar_ev[LEGISTAR_SESSION_TIME]
+            # better to return time as local time with time zone info,
+            # rather than as utc time.
+            # this way the calling pipeline can find out what is the local zone.
+            session_time = self.as_local_time(
+                self.date_time_to_datetime(
+                    legistar_ev[LEGISTAR_SESSION_DATE],
+                    legistar_ev[LEGISTAR_SESSION_TIME],
+                )
```

Comment:
For safety can you also localize the event_datetime. On the `EventIngestionModel`.

Reply:
Yes for sure. That's my bad. Missed that one.

Reply:
Sorry didn't see this edit. You are entirely correct. For some reason I thought there was an `event_datetime` property on the `EventIngestionModel` object. But yep it's entirely handled downstream so no worries here.